firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -111,22 +123,53 @@ public static FirebaseAppDistribution getInstance(@NonNull FirebaseApp app) {
    * to complete the download and installation.
    */
   @NonNull
-  public synchronized Task<Void> updateToLatestRelease() {
-    if (cachedUpdateToLatestReleaseTask != null && !cachedUpdateToLatestReleaseTask.isComplete()) {
+  public synchronized UpdateTask updateIfNewReleaseAvailable() {
+    synchronized (updateTaskLock) {
+      if (cachedUpdateToLatestReleaseTask != null
+          && !cachedUpdateToLatestReleaseTask.isComplete()) {
+        return cachedUpdateToLatestReleaseTask;
+      }
+
```

Comment:
I think we should rename this task `cachedUpdateIfNewReleaseAvailable` to reflect the change in the method name

Reply:
Yea most name changes will happen in an additional PR.